tendrl/gluster_integration/objects/brick/__init__.py

```diff
@@ -4,34 +4,60 @@
 class Brick(objects.BaseObject):
```

Comment:
We are not landing the Gluster Brick changes https://github.com/Tendrl/gluster-integration/pull/282 
until the UI folks ack on that, So please remove changes related to that and only keep the alert triggers